CHANGELOG.md

```diff
@@ -9,27 +9,31 @@ Notes on dependencies:
   rtree package has also been removed. (#3035)
 
 New methods:
-- Added `count_coordinates` method from shapely to GeoSeries/GeoDataframe (#3026).
-- Added `minimum_clearance` method from shapely to GeoSeries/GeoDataframe (#2989).
-- Added `is_ccw` method from shapely to GeoSeries/GeoDataframe (#3027).
+
+- Added ``count_coordinates`` method from shapely to GeoSeries/GeoDataframe (#3026).
```

Comment:
None of these quote changes should be needed after #2662, they all seem to be rendered fine as is for me?

Reply:
Ah, true! I forgot we no longer embed the markdown in RST. I will revert those.

Reply:
I've kept the empty lines. Those are not strictly required to properly render but are recommended by markdownlint.